src/iasm.c

```diff
@@ -426,14 +426,10 @@ static OPND *asm_and_exp();
 static OPND *asm_cond_exp();
 static opflag_t asm_determine_operand_flags(OPND *popnd);
 static code *asm_genloc(Loc loc, code *c);
-static int asm_getnum();
+static long long asm_getnum();
```

Comment:
I have a better idea. Since `asm_getnum` is only used in one place, and its output is immediately checked by `isPowerOf2`, we can just return `int(-1)` and the caller will reject it in any case (because `0xffffffff` is not a power of 2).